go/oasis-test-runner/scenario/e2e/runtime/byzantine.go

```diff
@@ -19,16 +19,17 @@ import (
 
 var (
```

Comment:
There is a `schedulerCheckScheduled` method in the byzantine worker that assumes distinct members in the `committee.Members` and should be updated for future proofness.

Reply:
Good catch, updated.